### ISSUE TITLE
[TimePicker] Japanese translation is not good

### ISSUE THREAD

**Issue by manabu-nakamura** [material-components/material-components-android#3861]
**Description:** The Japanese translation is not good.
![Screenshot_20231121_183813](https://github.com/material-components/material-components-android/assets/71050561/6c91cb35-7943-45f7-ad21-b83d58c81c8a)

**Expected behavior:**
![Screenshot_20231121_215158](https://github.com/material-components/material-components-android/assets/71050561/39768c55-2a1f-4950-bc4f-fd71d809a0bc)
(android.app.TimePickerDialog)
![Screenshot_20231121_184925](https://github.com/material-components/material-components-android/assets/71050561/3bcc9ab6-ba8c-4068-b0d9-7475c56b4eac)

**Minimal sample app repro:** Catalog app (TimePickerMainDemoFragment)

**Android API version:** 34

**Material Library version:** 1.12.0-alpha01

**Device:** Emulator 32.1.15 (API 32)

**Comment by manabu-nakamura** [comment]
一時的な解決策：これらをres/values-ja/strings.xmlに追加してください
```
<string name="material_timepicker_select_time">時刻を選択</string>
<string name="material_timepicker_hour">時</string>
<string name="material_hour_24h_suffix">%1$s時</string>
<string name="material_hour_selection">時を選択</string>
```

**Comment by pekingme** [comment]
Ah, I think there's a limitation to translate this from English to Japanese. In English, there's no different word representing the "hour" of a moment than the "hour" of a duration. I think it makes sense to add the custom strings as you suggested.

**Comment by manabu-nakamura** [comment]
I think it's a mistranslation.
Because the time picker is a component to choose not the duration of time but the moment of time.

**Comment by pekingme** [comment]
Yes, agree. I have reported this to our i18n team, who is in charge of the translations. Will keep this issue open until that's updated internally. Thanks for reporting and making the PR.
